.github/workflows/install.yaml

```diff
@@ -0,0 +1,35 @@
+name: Install Tests
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+jobs:
+  install:
+    name: ${{ matrix.os }} - ${{ matrix.python_version }} install
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
should we add windows as well?

Reply:
Doing just ubuntu for now to keep testing to a minimal